crates/gitbutler-reference/tests/reference.rs

```diff
@@ -5,11 +5,12 @@ mod normalize_branch_name {
     fn valid_substitutions() {
         for (input, expected) in [
             ("a", "a"),
-            ("a+b", "a-b"),
+            ("a+b", "a+b"),
             ("a^b", "a-b"),
+            ("a^^^b", "a-b"),
             ("a~b", "a-b"),
-            ("a<b", "a-b"),
```

Comment:
It's notable that I enlarged the allowed set of characters to be closer to what Git would allow, but that might be undesired.